### PR TITLE
Refactor consents routes, nest patients page

### PR DESCRIPTION
**Pull request by misaka**
This rejigs the routes so that we can nest the patients page under the
correct consents tab:

/sessions/:session_id/consents/:tab/patients/:patient_id

And also as a side-effect changes the generated helpers for the routes,
to something more sensible imo:

consents_session_path -> session_consents_path

### CODE REVIEW

File: app/components/app_consent_component.rb
Comment (tvararu) [inline app/components/app_consent_component.rb:47]: Hmm, does this work as-is? Maybe for consistency:

```suggestion
        session_id: session.id,
```

Or if the var was an id all along:

```suggestion
        session_id:,
```

File: app/controllers/manage_consents_controller.rb
Comment (tvararu) [inline app/controllers/manage_consents_controller.rb:95]: - session_consents_path
- triage_session_path
- session_vaccinations_path

Triage is the odd one out, how come? I thought it should be `session_triage_path` according to the new routes

File: app/controllers/patient_sessions_controller.rb
Comment (tvararu) [inline app/controllers/patient_sessions_controller.rb:54]: See above

File: app/controllers/vaccinations_controller.rb
Comment (tvararu) [inline app/controllers/vaccinations_controller.rb:94]:
```suggestion
                        patient_id: @patient.id
```

File: app/controllers/vaccinations_controller.rb
Comment (tvararu) [inline app/controllers/vaccinations_controller.rb:99]:
```suggestion
                        patient_id: @patient.id
```

File: app/controllers/vaccinations_controller.rb
Comment (tvararu) [inline app/controllers/vaccinations_controller.rb:110]:
```suggestion
                      patient_id: @patient.id
```

File: app/helpers/application_helper.rb
Comment (tvararu) [inline app/helpers/application_helper.rb:48]:
```suggestion
```

File: app/views/sessions/show.html.erb
Comment (tvararu) [inline app/views/sessions/show.html.erb:39]: Worth checking this in like a timeboxed ~5 minutes but I'd remove these TODOs before merging even if we don't. It's really not that bad having to specify it just in this instance.

File: config/initializers/tab_paths.rb
Comment (tvararu) [inline config/initializers/tab_paths.rb:23]:
```suggestion
```

**Review by tvararu — APPROVED** [review]
New routes and URLs are super clean 👍 

New path helpers also clearer

Back link works predictably based on a bit of testing

Left some comments but pre-approving pending commit tidy up, well done! 🙌

File: app/components/app_consent_component.rb
Comment (misaka) [reply app/components/app_consent_component.rb:47]: In my experimentation, `session_id` can be set to either `session` or `session.id`, so the original code above should work, ya. However, like you suggest, I've been trying to standardise on `session.id`, I probably just missed that one. It's a shame that `session: session` doesn't work as that could've been nicely abbreviated as `session:`

**Comment by misaka** [comment]
The comments are spot-on, and things that just haven't been changed because this is still a WIP. I'll update it, probably addressing most if not all the comments here, and then maybe you can have another look @tvararu . Cheers!

**Pull request merged**
